live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/RepoEventMapperTest.java

```diff
@@ -69,6 +70,33 @@ class RepoEventMapperTest
     private static final Set<NodeProperty<?>> NODE_PROPERTIES = Set.of(NODE_TITLE);
     private final RepoEventMapper repoEventMapper = new RepoEventMapper();
 
+    @Test
+    void shouldMapToIngestNodeContentCommand()
+    {
+        // given
+        RepoEvent<DataAttributes<NodeResource>> event = mock();
+        DataAttributes<NodeResource> data = mock();
+
+        when(event.getTime()).thenReturn(dateFromTimestamp(EVENT_TIMESTAMP));
```

Comment:
Maybe we could also use here the MockitoBDD's `given()` instead of `when()`?

Reply:
sure, fixed